pages/mint-nft/index.vue

```diff
@@ -196,6 +278,10 @@ const iscnIdInput = ref(route.query.class_id || route.query.iscn_id || '')
 const iscnOwner = ref('')
 const iscnCreateData = ref<any>(null)
 const iscnData = ref<any>(null)
+const nftIdPrefix = ref('BOOKSN')
```

Comment:
what does `BOOKSN` mean here?

Reply:
Book Serial Number.  This label is not important and should not hold the user back for even 1 second, so give it a default.